apps/browser/qml/pages/components/Overlay.qml

```diff
@@ -321,7 +321,11 @@ Shared.Background {
                 textTopMargin: height/2 - _editor.implicitHeight/2
                 labelVisible: false
                 inputMethodHints: Qt.ImhUrlCharactersOnly
-                background: null
+                background: Rectangle {
```

Comment:
Just as a note, this will conflict with the changes in #800.

Reply:
done